script/test

```diff
@@ -45,6 +45,7 @@ readonly THREAD_VERSION="${THREAD_VERSION:-1.1}"
 readonly INTER_OP="${INTER_OP:-0}"
 readonly VERBOSE="${VERBOSE:-0}"
 readonly BORDER_ROUTING="${BORDER_ROUTING:-1}"
+readonly INTER_OP_BBR="${INTER_OP_BBR:-1}"
```

Comment:
Add usage documentation for `INTER_OP_BBR`?

Reply:
Updated.